src/components/challenge/_.module.css

```diff
@@ -0,0 +1,87 @@
+.challenge {
```

Comment:
![image](https://user-images.githubusercontent.com/82516098/125385375-0b148280-e3a3-11eb-868b-3d924f008d84.png)
do not have these problems with footer?